sdk/storage/azure_storage_queue/src/generated/clients/queue_client.rs

```diff
@@ -350,7 +350,9 @@ impl QueueClient {
     /// async fn example() -> Result<()> {
     ///     let response: Response<QueueClientGetMetadataResult, NoFormat> = unimplemented!();
     ///     // Access response headers
-    ///     println!("x-ms-meta: {:?}", response.metadata()?);
+    ///     if let Some(metadata) = response.metadata()? {
```

Comment:
This will fail. You need to regenerate based on `main` now, which means you need to sync. We just merged the emitter change about 30 minutes ago that fixes this.